src/apps/common/submodules/mediaSelect/mediaSelect.css

```diff
@@ -78,3 +78,19 @@
 .monster-media-selection.monster-media-tabs-skin .upload-div {
 	margin-top: 0;
 }
+
+.monster-media-selection.monster-media-tabs-skin .monster-tab-content-tts .custom-greeting-text {
+	width: 100%;
+	box-sizing: border-box;
+	margin-bottom: 4px;
+}
+
+.monster-media-selection.monster-media-tabs-skin .monster-tab-content-tts .chars-remaining-counter {
+	font-size: 12px;
+	color: #64646c
+}
+
+.monster-media-selection.monster-media-tabs-skin .monster-tab-content-tts .chars-remaining-counter span {
+	font-weight: bold;
+	margin-right: 4px;
+}
```

Comment:
Please transition this sheet to scss so you can use nested rules and existing colors variables.

If you have time and since the amount of rules is pretty small, you can even refactor the whole sheet (as long as it's in its own specific "refactor" commit that it does not contain any other changes).